schema

```diff
@@ -241,16 +240,22 @@
             }
           },
           "anyOf": [
-            {"required": ["data"]},
-            {"required": ["meta"]},
-            {"required": ["links"]}
+            {
+              "required": [
+                "data"
+              ]
+            },
+            {
+              "required": [
+                "meta"
+              ]
+            },
+            {
+              "required": [
+                "links"
+              ]
+            }
           ],
-          "not": {
-            "anyOf": [
-              {"required": ["id"]},
-              {"required": ["type"]}
-            ]
-          },
```

Comment:
should this be moved up one level instead of removed to satisfy the restriction

> a resource can not have an attribute and relationship with the same name, **nor can it have an attribute or relationship named type or id.**